code/game/objects/items/melee/baton.dm

```diff
@@ -127,7 +127,7 @@
 
 	if(!chunky_finger_usable && ishuman(user))
 		var/mob/living/carbon/human/potential_chunky_finger_human = user
-		if(potential_chunky_finger_human.check_chunky_fingers() && user.is_holding(src) && !HAS_TRAIT(user.mind, TRAIT_CHUNKYFINGERS_IGNORE_BATON))
+		if(potential_chunky_finger_human.check_chunky_fingers() && user.is_holding(src) && !HAS_TRAIT(user, TRAIT_CHUNKYFINGERS_IGNORE_BATON) && !HAS_TRAIT(user.mind, TRAIT_CHUNKYFINGERS_IGNORE_BATON))
```

Comment:
This will still runtime, you will need to check that `user.mind` exists. 
```dm
if(potential_chunky_finger_human.check_chunky_fingers() \
    && user.is_holding(src) \
    && !HAS_TRAIT(user, TRAIT_CHUNKYFINGERS_IGNORE_BATON) \
    && (user.mind && !HAS_TRAIT(user.mind, TRAIT_CHUNKYFINGERS_IGNORE_BATON) \
)
```